oss-internship-2020/libuv/generator/wrapper_generator.py

```diff
@@ -0,0 +1,218 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script generating a wrapper API for LibUV."""
```

Comment:
Please mention that this is highly specific to libuv's source code and does not easily generalize.

Also add a blank line above.